### ISSUE TITLE
Create installer for ArcGIS version

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CEAforArcGIS#355]
I just checked all the libraries we depend on and they are all BSD, MIT or LGPL. I believe that means we can write an installer (`Setup.exe`) that will install the CEA for users without having to follow the instruction manual.

I have used a prototype of this for the MBS students (computer lab, laptops) that basically has them copy pre-prepared files and folders to the computer. This format is easy to bundle into an installer. We should do this.

**Comment by JIMENOFONSECA** [comment]
@daren-thomas i was wonder if we could target to get to install the CEA one day from pip install. or easy_install. Just as many other libraries of python that exist our there. 

I guess we can split the process in two things:

- one is the CEA as a library and 'engine'
- one is the installer of the plug-in to make it work with arcgis.

I believe both should be different. In this way we do not compromise researchers working with the engine to get necessarily into Arcgis (which has to be payed for).

**Comment by daren-thomas** [comment]
PR #376 is a first step: It includes an Inno Setup file that can create a file `Setup_CEAforArcGIS10.4.exe` that will install the CEA on a system. I have tested this on an empty VM that only contains ArcGIS 10.4 on Windows 10 and ran all the scripts (except radiation) and it works! This uses a simplified installation procedure that just copies the conda environment's `site-packages` folder to the user's computer, without actually installing Anaconda. I believe we are allowed to do this.

If this get's accepted, then we would need to address the following points as new individual issues:

- add a README file to the installation folder containing information about install locations (be a good citizen) and where to find more information
- add the PDF of the documentation with a link from the windows start-menu?
- add a LICENSE file to the installation folder, mentioning also the included libraries, there licenses and where their source can be downloaded
- an Icon for the installer instead of the standard setup icon :)
- document how to set up a developer machine to create the setup.exe

**Comment by daren-thomas** [comment]
@JIMENOFONSECA: Here is a link to the [Setup_CEAforArcGIS10.4.exe](https://www.dropbox.com/s/96gl5f3uxqq23s6/Setup_CEAforArcGIS10.4.exe?dl=0) if you want to test it.

**Comment by JIMENOFONSECA** [comment]
Great stuff.

would you mind including the next libraries? (my next pull request requires them, so it would be good to have them to test it).

- Salib
- plotly
- pydoe

thanks!

On 19 Oct 2016, at 8:41 pm, Daren Thomas <notifications@github.com<mailto:notifications@github.com>> wrote:


@JIMENOFONSECA<https://github.com/JIMENOFONSECA>: Here is a link to the Setup_CEAforArcGIS10.4.exe<https://www.dropbox.com/s/96gl5f3uxqq23s6/Setup_CEAforArcGIS10.4.exe?dl=0> if you want to test it.

**Comment by daren-thomas** [comment]
Hi Jimeno, this installer only installs the `cea` branch. plotly is included but I'd prefer not to include libraries "in advance". How about we merge the PR and then do this in a separate issue?

**Comment by daren-thomas** [comment]
I have opened a new issue #403 for the next milestone to deliver a `pip install cea` mechanism.

There are still some tasks that need to be completed before we can close this issue, see above.

**Comment by daren-thomas** [comment]
fixed in PR #429
